core-bundle/src/Resources/contao/classes/Theme.php

```diff
@@ -663,9 +723,56 @@ protected function extractThemeFiles($arrFiles, $arrDbFields)
 		$this->import(Automator::class, 'Automator');
 		$this->Automator->generateSymlinks();
 
+		if (($exampleWebsite = Input::post('example_website')) && isset($exampleWebsites[$exampleWebsite]))
+		{
+			$importType = Input::post('example_website_import');
+			$this->importExampleWebsite($exampleWebsites[$exampleWebsite], $importType === 'data_no_truncate', $importType !== 'full');
+		}
+
 		$this->redirect(str_replace('&key=importTheme', '', Environment::get('request')));
 	}
 
+	private function importExampleWebsite(string $exampleWebsite, bool $preserveData, bool $insertOnly): void
+	{
+		$connection = System::getContainer()->get('database_connection');
+
+		$userRow = $connection->fetchAssociative('SELECT * FROM tl_user WHERE id = ?', array((int) BackendUser::getInstance()->id));
+
+		if (!$preserveData && $insertOnly)
+		{
+			$tables = $connection->createSchemaManager()->listTableNames();
+
+			foreach ($tables as $table)
+			{
+				if (0 === strncmp($table, 'tl_', 3))
+				{
+					$connection->executeStatement('TRUNCATE TABLE ' . $connection->quoteIdentifier($table));
+				}
+			}
+		}
+
+		$data = file(Path::join(System::getContainer()->getParameter('kernel.project_dir'), $exampleWebsite));
+
+		try
+		{
+			foreach ($insertOnly ? preg_grep('/^INSERT /', $data) : $data as $query)
```

Comment:
I think this would fail if there are no `INSERT` statements

Reply:
Cannot confirm this. How can I reproduce that?

Reply:
ah I was confusing the method. But would that mean we do not support multiline SQL statements? Did we also not support that previously?

Reply:
Correct, multiline is not supported, same as it was in the install tool.